frontend/src/components/App/Notifications/List.stories.tsx

```diff
@@ -17,9 +16,13 @@ function createNotifications() {
         date: '2022-08-01',
       })
     );
+
+    if (i % 10 === 0) {
+      notifications[i].cluster = 'cluster';
```

Comment:
umm I don't understand this change

Reply:
Every 10th one has a cluster field set to 'cluster'. So it shows some different data in there.